monitor/src/js/app.js

```diff
@@ -0,0 +1,101 @@
+let headers = new Headers();
+
+const key = 'Basic SFNlcnZpY2U6TmU1bWkxdnU=';
```

Comment:
Shouldn't be here

Reply:
just ask user for input, and save it to session storage, but actual credentials should be removed form github history - they are public